skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevvelger/$templateId/-components/endreMottaker/EndreMottakerUtils.ts

```diff
@@ -107,14 +107,46 @@ export const leggTilManuellSamhandlerFormDataSchema = z.object({
       }
       return refinementContext;
     }),
-  adresse: z.object({
-    navn: z.string().min(1, "Obligatorisk"),
-    linje1: z.string().min(1, "Obligatorisk"),
-    linje2: z.string(),
-    postnr: z.string().min(1, "Obligatorisk"),
-    poststed: z.string().min(1, "Obligatorisk"),
-    land: z.string().min(1, "Obligatorisk"),
-  }),
+  /**
+   * manuell adresse har 2 krav for utfylling:
+   * For norske adresser (der land er "NO") - kreves navn, postnummer, og poststed.
+   * For utenlandkse adresser (alle andre land som ikke er "NO") - kreves navn, og adresselinje1
+   */
+  adresse: z
+    .object({
+      navn: z.string().min(1, "Obligatorisk"),
+      linje1: z.string(),
+      linje2: z.string(),
+      postnr: z.string(),
+      poststed: z.string(),
+      land: z.string().min(1, "Obligatorisk"),
+    })
+    .superRefine((data, refinementContext) => {
+      if (data.land === "NO") {
```

Comment:
Det at `land` er `"NO"` er kun for frontend sant? Backend sender nemlig ikke med land for norske adresser.

Reply:
I frontend så henter vi land fra backend hvis dem skal fylle ut adressen manuellt. Derimot så er det slik at defualt valg skal være Norge, som er "NO" i iso-standaren. Dette er teknisk sett en quick and dirty måte å kunne gjøre validering på. Er litt usikker på hvordan vi ellers kan validere land-verdien mot